travis/before_install.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Don't source setup.sh here, because the virtualenv might not be set up yet
+
+# Install dependencies
+# removing the old c++ link is needed because otherwise brew install fails
+rm '/usr/local/include/c++'
+brew install ccache protobuf gcc@5
+
+# Install Python.
+# The brew update/upgrade commands are needed because otherwise homebrew
+# exits with an error code when installing python2 since it is already installed
+# but with a different version.
+if [ "${PYTHON_VERSION}" != "python2" ]; then
+  brew install ${PYTHON_VERSION}
```

Comment:
IIRC the builtin python2 version is pretty old.

Reply:
It's actually 2.7.14, that's up to date.

Reply:
Doesn't sound right, could you check whether it's travis has pre installed brew python2 for you? (do brew info python2 at the beginning.)

Reply:
I checked in a CI log. Without any changes, this CI setup uses python 2.7.14 in the virtualenv. There's an output for the python version in setup.sh.